src/AcceptanceTests/Shared/Request_reply.cs

```diff
@@ -11,7 +12,14 @@ public async Task Should_get_the_reply()
     {
         var ctx = await Scenario.Define<Context>()
                     .WithEndpoint<SendingEndpoint>(c => c
-                        .When(b => b.Send(new MyMessage())))
+                        .When(cc => cc.EndpointsStarted, (b, _) =>
+                        {
+                            var sendOptions = new SendOptions();
+
+                            sendOptions.RouteReplyTo(Conventions.EndpointNamingConvention(typeof(SendingEndpoint)));
```

Comment:
This shouldn't be needed since the reply to address by default  uses the endpoint name which in turn should be using the endpoint naming convention. This could be verified by checking that the reply to header has the same value in both cases?